lib/src/game/base_game.dart

```diff
@@ -2,15 +2,11 @@ import 'dart:ui';
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
-import 'package:flutter/widgets.dart' hide WidgetBuilder;
 import 'package:ordered_set/comparing.dart';
 import 'package:ordered_set/ordered_set.dart';
 
-import '../components/component.dart';
-import '../components/mixins/has_game_ref.dart';
-import '../components/mixins/tapable.dart';
-import '../components/position_component.dart';
-import '../extensions/vector2.dart';
+import '../../components.dart';
+import '../../extensions.dart';
```

Comment:
Do we know if that affects tree shaking?

Reply:
I will revert for now until we have renan's answer

Reply:
cc @renancaraujo